app/src/main/java/org/schabi/newpipe/util/SecondaryStreamHelper.java

```diff
@@ -27,12 +30,14 @@ public SecondaryStreamHelper(@NonNull final StreamInfoWrapper<T> streams,
     /**
      * Find the correct audio stream for the desired video stream.
      *
+     * @param context      Android context
      * @param audioStreams list of audio streams
      * @param videoStream  desired video ONLY stream
      * @return selected audio stream or null if a candidate was not found
      */
```

Comment:
```suggestion
    /**
     * Finds an audio stream compatible with the provided video-only stream, so that the two streams
     * can be combined in a single file by the downloader. If there are multiple available audio
     * streams, chooses either the highest or the lowest quality one based on
     * {@link ListHelper#isLimitingDataUsage(Context)}.
     *
     * @param context      Android context
     * @param audioStreams list of audio streams
     * @param videoStream  desired video-ONLY stream
     * @return the selected audio stream or null if a candidate was not found
     */
```